docs/src/modules/components/AppSearch.js

```diff
@@ -192,6 +192,7 @@ class AppSearch extends React.Component {
           <SearchIcon />
         </div>
         <Input
+          aria-label="Search in Material-UI documentation"
```

Comment:
https://github.com/algolia/docsearch/issues/418

Reply:
I think that it should be:
```jsx
        <Input
          inputProps={{
            'aria-label': "Search in Material-UI documentation",
          }}
```
But then, Algolia removes the label, we might have to do it with a use effect.

Reply:
:disappointed:

Reply:
Naively thought this would be enough. Then tried `inputProps` instead. Thanks for saving me time debugging this :+1: